JSON_Schemas/Weights_and_Condition_Scores/iscFatScore.json

```diff
@@ -0,0 +1,46 @@
+{
+    "required": [
+    ],
+
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "iscScoreEventCore.json"
+        },
+        {
+            "properties": {
+                "fatScoreReference": {
+                    "type": "object",
+                    "$ref": "iscScoreReference.json",
+                    "description": "The scheme or scoring reference guideline being used"
+                },
+                "score": {
+                    "type": "number",
+                    "description": "Value of the fat score"
+                },
+                "fatDepths": {
+                    "type": "array",
+                    "objects": {
+                        "type": "object",
+                        "properties": {
+                            "bodySite": {
+                                "type": "string",
+                                "description": "Body Site where the measurement was taken"
+                            },
+                            "fatDepth": {
+                                "type": "number",
+                                "format": "double",
+                                "description": "Usually measure in Millimeters"
```

Comment:
Description - usually "measured'' in Millimeters

Reply:
I thought we had agreed that Fat Depth was not part of fat score and was in fact a separate measure and not a score at all.

Reply:
Removed the fat depths from iscFatScore